### PR TITLE
refactor(LTE): add generic NetworkManager config and some documentation

### PR DESCRIPTION
**Pull request by HarryWaschkeit**
- added a generic NetworkManager configuration to automatically connect via cellular modem (LTE normally)
- added general documentation about LTE on omnect OS

### CODE REVIEW

File: doc/LTE.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# LTE Support
+
+While omnect OS is generally prepared for LTE support on respective
+devices, for it to to really work there can be some extra configuration
```
Comment (JanZachmann) [inline doc/LTE.md:4]: typo "to to"

**Review by JanZachmann — CHANGES_REQUESTED** [review]

File: doc/LTE.md (hunk 1 above)
Comment (HarryWaschkeit) [reply doc/LTE.md]: fixed.

File: doc/WELOTEC.md [hunk 2]
```diff
@@ -18,4 +18,6 @@
 | **OTA Update**                       | yes             |
 | **Factory Reset**                    | yes             |
 | **Wifi Commissioning via Bluetooth** | no              |
-| **LTE Support**                      | experimental    |
+| **LTE Support** [^1]                 | experimental    |
+
+[^1]: in [LTE documentation](LTE.md) you can find more details concerning configuration
```
Comment (JoergZeidler) [inline doc/WELOTEC.md:23]: do you have a idea why a link symbol at the end of the line is present?

Comment (JoergZeidler) [reply doc/WELOTEC.md:23]: see: https://github.com/HarryWaschkeit/meta-omnect/blob/hwt-lte-default-config-and-docu/doc/WELOTEC.md#user-content-fnref-1-bca1608a12dcceb80942b7ef9e0fef1d

Comment (HarryWaschkeit) [reply doc/WELOTEC.md:23]: this is standard markdown behavior (at least of github) for footnotes, maybe to have a perma-link also at hand in the footnote itself.

**Review by JoergZeidler — APPROVED** [review]

**Review by mlilien — APPROVED** [review]

**Review by JanZachmann — APPROVED** [review]

**Pull request merged**